plugins/discovery-file/build.gradle

```diff
@@ -27,3 +31,36 @@ bundlePlugin {
         into 'config'
     }
 }
+
+task seedTask << { }
+// setup the initial cluster with one node that will serve as the seed node
+// for unicast discovery
+ClusterConfiguration config = new ClusterConfiguration(project)
+config.distribution = 'zip'
```

Comment:
We should not be using the full zip here. Integration tests for plugins should be isolated from other plugins/modules.